packages/olo-gulp-helpers/package.json

```diff
@@ -18,7 +18,7 @@
     "gulp-if": "^2.0.0",
     "gulp-plumber": "^1.1.0",
     "gulp-rev": "^7.0.0",
-    "gulp-sass": "^2.2.0",
+    "gulp-sass": "3.1.0",
```

Comment:
Intentional to lock this to a specific version instead of a range like the rest? Not necessarily opposed, just wondering if intentional

Reply:
locked because I won't be monitoring those builds as closely as Dashboard and wary of breaking changes making it into a minor release

happy to change it if you recommend